examples/blender/blender.py

```diff
@@ -85,17 +85,21 @@ async def worker(ctx: WorkContext, tasks):
     parser.set_defaults(log_file="blender-yapapi.log")
     args = parser.parse_args()
 
+    asyncio_fix()
```

Comment:
Why not apply the fix automatically? E.g. by calling `asyncio_fix()` in `yapapi/__init__.py`?

Reply:
hmm... I dunno - I'd rather have it called explicitly - especially since this is a library and maybe people will need to have it cooperate with other async code ...

Reply:
I think a more common scenario (than the one in which us setting a particular implementation of event loop causes problems with advanced asyncio usage) is that a Windows user forgets to import or call `fix_asyncio()`.

Reply:
Windows users have to deal with default implementation of event loop being different in Python 3.6/3.7 and Python 3.8/3.9 anyway.